react/Autosuggest/Autosuggest.demo.js

```diff
@@ -49,6 +49,15 @@ class AutosuggestContainer extends Component {
           }}
           onClear={this.handleClear}
         />
+        <DemoComponent
```

Comment:
I don't want to have it in demo at all being hones, just for demonstration purpose, will merge without it